-- Fix PEP 621 license field for the setuptools version in ports.
-- Old setuptools rejects the shorthand "license = \"MIT\"" syntax.

--- pyproject.toml.orig	2026-09-11 19:44:05 UTC
+++ pyproject.toml
@@ -5,7 +5,7 @@ version = "1.43.93"
 [project]
 name = "boto3-stubs"
 version = "1.43.93"
-license = "MIT"
+license = {text = "MIT"}
 authors = [{ name = "Vlad Emelianov", email = "vlad.emelianov.nz@gmail.com" }]
 description = "Type annotations for boto3 1.43.93 generated with mypy-boto3-builder 8.12.0"
 readme = "README.md"
